sfs/util.py

```diff
@@ -487,6 +487,32 @@ def _count_walls_1d(a):
     return xs, wall_count
 
 
+def max_order_circular_harmonics(N, max_order):
+    r"""Maximum order of 2D/2.5D HOA.
+
+    If ``max_order`` is ``None`` it returns the maximum order
+    for which no spatial aliasing appears.
+    It is given on page 132 of [Ahrens2012]_ as
+
+    .. math::
+        \text{max_order} =
+            \begin{cases}
+                N/2 - 1 & \text{even}\;N \\
+                (N-1)/2 & \text{odd}\;N
+            \end{cases}
```

Comment:
Since you are using a different (but apparently equivalent) equation in the code, you should mention this here.